### DISCUSSION TITLE
Old forecast data returned

### DISCUSSION THREAD

**Discussion by vestmon** [weather-gov/api#292] [Q&A]
It appears the something is caching endpoints.  This problem started about two weeks ago.  When I retrieve an endpoint that has been previously retrieved, I receive old data. I have tried adding a Cache-Control header to the call to no avail.  If I fetch different gridpoints, there is not an issue.  The normal cache busting techniques like adding a arbitrary argument do not work as the resulting url is invalid.  

# curl -i https://api.weather.gov/gridpoints/BMX/110,42/forecast HTTP/1.1 200 OK
Server: nginx/1.16.1
Content-Type: application/geo+json
Last-Modified: Mon, 28 Sep 2020 18:00:00 GMT
Access-Control-Allow-Origin: *
Access-Control-Allow-Headers: Feature-Flags
X-Request-ID: 8056ca29-edbb-4663-9d32-fb8130a25615
X-Correlation-ID: 6eb39977
X-Server-ID: vm-lnx-nids-apiapp6.ncep.noaa.gov
Cache-Control: public, max-age=282, s-maxage=3600
Expires: Sun, 06 Sep 2020 17:50:58 GMT
Date: Sun, 06 Sep 2020 17:46:16 GMT
Content-Length: 12641
Connection: keep-alive
X-Edge-Request-ID: c457a7e
Vary: Accept,Feature-Flags,Accept-Language
Strict-Transport-Security: max-age=31536000 ; includeSubDomains ; preload

{
    "@context": [
        "https://geojson.org/geojson-ld/geojson-context.jsonld",
        {
            "@version": "1.1",
            "wx": "https://api.weather.gov/ontology#",
            "geo": "http://www.opengis.net/ont/geosparql#",
            "unit": "http://codes.wmo.int/common/unit/",
            "@vocab": "https://api.weather.gov/ontology#"
        }
    ],
    "type": "Feature",
    "geometry": {
        "type": "Polygon",
        "coordinates": [
            [
                [
                    -85.501522699999995,
                    32.520713899999997
                ],
                [
                    -85.503401199999999,
                    32.4981376
                ],
                [
                    -85.476626499999995,
                    32.496550900000003
                ],
                [
                    -85.474742699999993,
                    32.519127000000005
                ],
                [
                    -85.501522699999995,
                    32.520713899999997
                ]
            ]
        ]
    },
    "properties": {
        "updated": "2020-08-24T11:08:56+00:00",
        "units": "us",
        "forecastGenerator": "BaselineForecastGenerator",
        "generatedAt": "2020-08-24T13:01:05+00:00",
        "updateTime": "2020-08-24T11:08:56+00:00",
        "validTimes": "2020-08-24T05:00:00+00:00/P8DT6H",

```

```

**Comment by JJKraw** [comment]
Have you looked at this thread?

https://github.com/weather-gov/api/issues/71

**Comment by vestmon** [comment]
It appears to be the same issue.
